test/reduceIndexed.js

```diff
@@ -101,4 +60,8 @@ describe('reduceIndexed', function() {
       );
     });
   });
+
+  it('should be curried', function() {
+    assert.strictEqual(RA.reduceIndexed(addition)(0)([1, 2, 3, 4]), 10);
```

Comment:
Let's add some more assertions. Thanks!
```suggestion
    assert.strictEqual(RA.reduceIndexed(addition, 0, [1, 2, 3, 4]), 10);
    assert.strictEqual(RA.reduceIndexed(addition)(0, [1, 2, 3, 4]), 10);
    assert.strictEqual(RA.reduceIndexed(addition, 0)([1, 2, 3, 4]), 10);
    assert.strictEqual(RA.reduceIndexed(addition)(0)([1, 2, 3, 4]), 10);
```